pages/user guides/instance_recommendation.md

```diff
@@ -0,0 +1,64 @@
+---
+toc: false
+title: Cloud Instance Recommendations
+keywords: vm, ec2, instance, cloud, recommendation, cpu usage
+last_updated: September, 2024
+summary: "How to generate cloud instance recommendations"
+sidebar: main_sidebar
+folder: userguides
+permalink: instance-recommendation.html
+---
+
+Tidal Accelerator analyzes your servers and generates data-driven cloud instance recommendations which meet performance requirements for your workloads at the best possible value.
+
+Instance recommendation is currently available for Azure and Amazon Web Services.
+
+## How does it work?
+
+Tidal Accelerator's cloud instance recommendation uses a proven scientific approach to compare the CPU performance of your on-premise servers with their cloud compute equivalent. This allows us to recommend instances which will meet performance requirements for your workloads while delivering the best possible value, ensuring that workloads are not over-provisioned. 
+
+We consider CPU age and processing power, as well as RAM and storage usage to generate our recommendations. As you discover more about your server infrastructure, your instance recommendations change and become more refined, based on real-world usage data. 
+
+Under the hood, we leverage the approach described in Amazon Web Service's whitepaper <a target="_blank" href="https://s3.ca-central-1.amazonaws.com/tidal.assets/aws-demystifying-vcpus.pdf">Demystifying the Number of vCPUs for Optimal Workload Performance</a>.
+
+This scientifically proven methodology drives the CPU performance aspect of our recommendations.
```

Comment:
```suggestion
This proven methodology drives the CPU performance aspect of our recommendations.
```

Reply:
I'm using the language from the whitepaper here but happy to remove this if it doesn't sound right
![image](https://github.com/user-attachments/assets/8a6aa51a-ca3f-43b3-86f9-a1e18cf1adcc)

Reply:
Great, if you're quoting them, that's fine. I _disagree_ with their assertion it's scientific, but as long as we aren't making that claim it's all good :)